Week2/homework/maartjes-work.js

```diff
@@ -46,16 +46,19 @@ const maartjesTasks = monday.concat(tuesday);
 const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
-  // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
+  return tasks
+    .map(task => task.duration / 60)
+    .filter(hour => hour >= 2)
+    .reduce((sum, hour) => sum + hour * hourlyRate, 0);
 }
 
 // eslint-disable-next-line no-unused-vars
 const earnings = computeEarnings(maartjesTasks, maartjesHourlyRate);
 
 // add code to convert `earnings` to a string rounded to two decimals (euro cents)
+const stringTypeOfEarning = earnings.toFixed(2).toString();
```

Comment:
`.toString()` is not needed here `.toFixed()` already produces a string.
Although the name `stringTypeOfEarning` is clear it's also a bit longer than necessary. I would call it something like `formattedEarnings`.

Reply:
Okay thanks for feedback. I didn't know that .toFixed already produces a string. And I thought that in the homework  We need to convert specified result to string.